### ISSUE TITLE
Wrong ES5 transpiling of v5 after #2823

### ISSUE THREAD

**Issue by oziniak** [erikras/redux-form#2825]
### Are you submitting a **bug report** or a **feature request**?

Bug report

### What is the current behavior?

When transpiling, `lib/createPropTypes` throws an error:
```
npm r..erver /home/oleh/8sph/react_bump_seg_fe/node_modules/redux-form/lib/createPropTypes.js:26
npm r..erver     errors: object, // a map of errors corresponding to structure of form data (result of validation)
npm r..erver             ^
npm r..erver ReferenceError: object is not defined
npm r..erver     at createPropTypes (/home/oleh/8sph/react_bump_seg_fe/node_modules/redux-form/lib/createPropTypes.js:26:13)
npm r..erver     at createAll (/home/oleh/8sph/react_bump_seg_fe/node_modules/redux-form/lib/createAll.js:135:46)
...
```
Chunk of code from  `lib/createPropTypes`:
```
...
var _propTypes = require('prop-types');
var _propTypes2 = _interopRequireDefault(_propTypes);
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
...
var obj = _propTypes2.default.obj;


var createPropTypes = function createPropTypes() {
  return {
    // State:
 ...
    errors: object, // a map of errors corresponding to structure of form data (result of validation)
```

### What is the expected behavior?
line 15 of `lib/createPropTypes` should transpile to `object = _propTypes2.default.object`

### What's your environment?

Redux form:`redux-form@^5.3.5`
system: `Mint 18 x86_64`
nodeVersion: `7.7.4`

**Comment by gustavohenke** [comment]
It seems like this is a duplicate of #2824, so I'll close it.
